web/resources/js/transaction-form.js

```diff
@@ -63,4 +63,27 @@ $(function () {
     });
     // Show add button in last row
     split_rows.last().find(".split-add-button").removeClass("hidden");
+
+  document.querySelector("#accept_selected").addEventListener('click', function (element){
+      var ids = [];
+        $('#transactions_unconfirmed').bootstrapTable('getSelections').forEach(function (element){
+            ids.push(element["id"]);
+        })
+      if(ids.length > 0)
+      $.ajax({
+          type: "POST",
+          url: "/finance/transaction/confirm_selected",
+          data: {ids: ids},
+          success: function (result) {
+             console.log(result);
+          },
+          error:function (error){
+              console.log(error);
+          },
+          dataType: "json"
+        });
+      setTimeout(() => { window.location.reload(); }, 500);
+  })
```

Comment:
It would be nice if you could try to avoid using jquery and use the more modern (and, more importantly, builtin) [`fetch` API](https://developer.mozilla.org/en-US/docs/Web/API/fetch).

See e.g. this sipa code to get a feeling for how it works: https://github.com/agdsn/sipa/blob/cdd8d40a977c3ea2c70d491ec9e56b2d8935946d/sipa/static/js/statuspage.js#L47-L57

Reply:
this only concerns the AJAX part, I'm aware that bootstrap-table requires access via jquery